src/core/MOM_stoch_eos.F90

```diff
@@ -66,22 +67,22 @@ subroutine MOM_stoch_eos_init(G,Time,param_file,stoch_eos_CS,restart_CS,diag)
   vd = var_desc("stoch_eos_pattern","nondim","Random pattern for stoch EOS",'h','1')
   call register_restart_field(stoch_eos_CS%pattern, vd, .false., restart_CS)
   ALLOC_(stoch_eos_CS%phi(G%isd:G%ied,G%jsd:G%jed)) ; stoch_eos_CS%phi(:,:) = 0.0
-  ALLOC_(l2_inv(G%isd:G%ied,G%jsd:G%jed)) ! currently allocating for data domain, should I which to compute domain?
-  ALLOC_(rgauss(G%isd:G%ied,G%jsd:G%jed)) ! currently allocating for data domain, should I which to compute domain?
+  ALLOC_(l2_inv(G%isd:G%ied,G%jsd:G%jed)) 
+  ALLOC_(rgauss(G%isd:G%ied,G%jsd:G%jed)) 
   call get_param(param_file, "MOM", "SEED_STOCH_EOS", seed, &
                  "Specfied seed for random number sequence ", default=0)
   call random_2d_constructor(rn_CS, G%HI, Time, seed)
   call random_2d_norm(rn_CS, G%HI, rgauss)
   ! fill array with approximation of grid area needed for decorrelation
   ! time-scale calculation
-  do j=G%jsd,G%jed
-     do i=G%isd,G%ied
+  do j=G%js,G%je
+     do i=G%is,G%ie
         l2_inv(i,j)=1.0/(G%dxT(i,j)**2+G%dyT(i,j)**2)
      enddo
   enddo 
   if (is_new_run(restart_CS)) then
-     do j=G%jsd,G%jed
-        do i=G%isd,G%ied
+     do j=G%js,G%je
```

Comment:
Are `js`, `je`, `is`, and `ie` defined somewhere? Or should these all have `c` at the end

Reply:
You are correct.  I do too much coding in the atmosphere model, where the is, js,etc. are defined.  
I made the fix.